drizzlepac/haputils/svm_quality_graphics.py

```diff
@@ -373,11 +424,189 @@ def build_crossmatch_plots(xmatchCDS, data_cols, output_basename='svm_qa'):
     save(column(plot_list))
 
     return output
+"""
+
+def build_interfilter_crossmatch_plots(xm_df, data_cols, display_plot, output_basename='svm_qa'):
+    """"Generate plots to statiscially quantify the quality of the alignment of filter-level HAP imagery
+    products
+
+    Parameters
+    ----------
+    xm_df : Pandas dataframe
+        This dataframe contains all the columns relevant to the plots.
+
+    data_cols : list
+        A subset of the column names in gaiaDF
+
+    output_basename : str
+        String to use as the start of the filename for the output plot pages.
+
+    display_plot : bool
+        Option to display the plot in addition to writing out the file.
+
+    Returns
+    -------
+    output : str
+        Name of HTML file where the plot was saved.
     """
+    # Setup the source of the data to be plotted so the axis variables can be
+    # referenced by column name in the Pandas dataframe
+    xm_cds = ColumnDataSource(xm_df)
+    num_of_datasets = len(xm_cds.data['index'])
+    print('Number of datasets: {}'.format(num_of_datasets))
+
+    output_basename = "{}_interfilter_crossmatch_comparison".format(output_basename)
```

Comment:
We are already generating filenames that are really long.  Perhaps something a little more concise would be just as good here; maybe just "interfilter_comparison" instead.

Reply:
issue resolved in commit [f852c01](https://github.com/spacetelescope/drizzlepac/pull/741/commits/f852c0190cb6a0f278e55f5c08d02beebc56a123)